tests/sentry/api/endpoints/test_auth_merge_user_accounts.py

```diff
@@ -0,0 +1,40 @@
+from sentry.testutils.cases import APITestCase
+from sentry.testutils.silo import control_silo_test
+
+
+@control_silo_test
+class ListUserAccountsWithSharedEmailTest(APITestCase):
+    endpoint = "sentry-api-0-auth-merge-accounts"
+    method = "get"
+
+    def test_simple(self):
+        user1 = self.create_user(username="mifu1", email="mifu@email.com")
+        user2 = self.create_user(username="mifu2", email="mifu@email.com")
```

Comment:
```suggestion
        user1 = self.create_user(username="mifu1", email="mifu@example.com")
        user2 = self.create_user(username="mifu2", email="mifu@example.com")
```

email.com is a real place, could we use example.com as it isn't a real place.